Makefile.am

```diff
@@ -6694,6 +6698,21 @@ tests += \
 	test-libsystemd-sym \
 	test-libudev-sym
 
+.PHONY: install-tests
+install-tests: $(tests) $(TEST_DATA_FILES)
```

Comment:
I think this PR is safe wrt. that. Regardless of whether you enable or disable unsafe tests, they don't get installed. Check this in Makefile.am:
```
noinst_PROGRAMS = $(manual_tests) $(tests) $(unsafe_tests)
TESTS = $(tests)
if ENABLE_UNSAFE_TESTS
TESTS += \
……………………$(unsafe_tests)
endif
```
but the `install-tests` target only works over `$(tests)`, *not* `$(TESTS)`.

That said, I'm fine with installing those into `$(testsdir)/{unsafe,manual}` in addition (or that could be a separate PR).